src/lib/components/SnoozePopup.js

```diff
@@ -83,3 +83,8 @@ export default class SnoozePopup extends React.Component {
     }
   }
 }
+
+SnoozePopup.propTypes = {
```

Comment:
Got a bunch more functions we could add here, [as it's rendered from `snooze-popup.js`](https://github.com/bwinton/SnoozeTabs/blob/master/src/popup/snooze-content.js#L100)